docs/doc_tutorial/advanced/training_seo.rst

```diff
@@ -0,0 +1,135 @@
+---
+SEO
+---
+
+ViUR offers the possibility to make URLs SEO-friendly in multiple languages.
+For this you can define seo-identifier for each url component.
+
+
+Module
+------
+The modulename is the first part of an URL.
+SEO-Identfiers has to be set as class-attribute ``seoLanguageMap`` of type ``dict[str, str]`` in the module.
+It maps a *language* to the acording *identifier*.
+
+.. code-block:: python
+    :name: module seo-map
+    :caption: modules/myorders.py
+    :emphasize-lines: 4-7
+
+    from viur.core.prototypes import List
+
+    class MyOrders(List):
+        seoLanguageMap = {
+            "de": "bestellungen",
+            "en": "orders",
+        }
+
+By default the module would be available under */myorders*, the lowercase modulename.
```

Comment:
```suggestion
By default, the module would be available under */myorders*, the lowercase module-name.
```

Reply:
I don't think the comma is correct there. It would be if it was a German sentence...